src/System.Runtime.CompilerServices.Unsafe/ref/Configurations.props

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netstandard1.0;
+      netstandard;
```

Comment:
the netstandard one will.

What do you mean by "crazy" dependencies?  Currently they're both building against System.Runtime, but the "netstandard" one should eventually build against netstandard.dll (once ilasm is fixed).  Both System.Runtime.dll and netstandard.dll are part of netcoreapp so I don't see either of them as "crazy".